ocaml/xapi/xapi_db_upgrade.ml

```diff
@@ -106,11 +106,6 @@ let jura =
   , Datamodel_common.jura_release_schema_minor_vsn
   )
 
-let yangtze =
```

Comment:
Should this be removed?

Reply:
We probably should keep this. Will fix it.